app/frontend/src/index.tsx

```diff
@@ -4,10 +4,13 @@ import { Provider } from "react-redux";
 import { PersistGate } from "redux-persist/integration/react";
 import "./index.css";
 import { persistor, store } from "./store";
+import * as api from "./service/api";
 
 import App from "./App";
 import * as serviceWorker from "./serviceWorker";
 
+api.setGetToken(() => store.getState().auth.authToken);
```

Comment:
Why is this necessary? The app has just loaded at this point so there won't be an auth token anyway?

Reply:
It is a function, so the api can always retrieve the latest version.